packages/kbn-apm-synthtrace-client/src/lib/apm/instance.ts

```diff
@@ -70,7 +70,6 @@ export class Instance extends Entity<ApmFields> {
       ...this.fields,
       'error.type': 'crash',
       'error.exception': [{ message, ...(type ? { type } : {}) }],
-      'error.grouping_name': getErrorGroupingKey(message),
```

Comment:
I found out that this is  causing the following issue
```
error": {
    "type": "document_parsing_exception",
    "reason": "[1:446] failed to parse field [error.grouping_name] of type [keyword] in document with id 'Bd2CBpMB-u5CWjDqauRY'. Preview of field's value: 'test'",
    "caused_by": {
      "type": "illegal_argument_exception",
      "reason": "Cannot index data directly into a field with a [script] parameter"
    }
  },
  
  
```
and as a result the documents are dropped.  

The `grouping_name` in the mapping

```
"grouping_name": {
              "type": "keyword",
              "script": {
                "source": """def logMessage = params['_source'].error?.log?.message;
if (logMessage != null && logMessage != "") {
  emit(logMessage);
  return;
}
def exception = params['_source'].error?.exception;
if (exception != null && exception.isEmpty() == false) {
  def exceptionMessage = exception instanceof Map ? exception?.message : exception[0]?.message;
  if (exceptionMessage instanceof List) {
    exceptionMessage = exceptionMessage[0]
  }
  if (exceptionMessage != null && exceptionMessage != "") {
    emit(exceptionMessage);
  }
}
""",
                "lang": "painless"
              }
            },
```

Reply:
So I removed it